tools/verify_headers.py

```diff
@@ -73,14 +73,15 @@ def validate_header(file_path):
 
 
 def main():
-    default_path = os.path.join(
+    default_paths = [os.path.join(
         os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
-        'qiskit_experiments')
+        'qiskit_experiments'),
+                     os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+        'test')]
```

Comment:
It might be a bit clearer to refactor this as:

```python
root_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
default_paths = [os.path.join(root_dir, 'qiskit_experiments), os.path.join(root_dir, 'test')]
```

Reply:
Sure, it probably doesn't hurt to lint the tools code